presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -84,6 +88,23 @@ public HiveTableProperties(TypeManager typeManager, HiveClientConfig config)
                                 .map(name -> ((String) name).toLowerCase(ENGLISH))
                                 .collect(Collectors.toList())),
                         value -> value),
+                new PropertyMetadata<>(
+                        ORC_BLOOM_FILTER_COLUMNS,
+                        "orc bloom index columns",
```

Comment:
"ORC Bloom filter index columns"

(it is capitalized on https://en.wikipedia.org/wiki/Bloom_filter )